### DISCUSSION TITLE
work with gltf extensions from cpp code

### DISCUSSION THREAD

**Discussion by noseeevil** [google/filament#4574] [Q&A]
Hello!

How can I change "KHR_texture_transform" properties of loaded GLB model from c++ code?
it is possible?

**Comment by prideout** [comment]
If your material provider is UbershaderLoader, you can look at ubershader.mat.in for the names of the parameters, e.g. `baseColorUvMatrix`, and call `setParameter(...)` on the material instance from C++.

If you are using MaterialGenerator instead of UbershaderLoader, the names are similar but there is a gotcha, which is that the material needs to use KHR_texture_transform upon load, otherwise the generated material will not support them.
